library/src/main/java/com/ogaclejapan/smarttablayout/SmartTabLayout.java

```diff
@@ -206,6 +206,17 @@ public void setCustomTabColorizer(TabColorizer tabColorizer) {
         mTabStrip.setCustomTabColorizer(tabColorizer);
     }
 
+
+    /**
+     * Sets the colors used for styling the tab text. This will need to be called prior to calling
+     * {@link #setViewPager(android.support.v4.view.ViewPager)} otherwise it will not get set
+     *
+     * @param tabTextColors ColorStateList to use for tab text
+     */
+    public void setTabTextColors(ColorStateList tabTextColors) {
```

Comment:
I'm sorry, could you change the method name?

```java
public void setDefaultTabTextColor(ColorStateList colors) {
    //....
}
```

I want to define a standard of TextView in the same way :D
http://developer.android.com/reference/android/widget/TextView.html#setTextColor(android.content.res.ColorStateList)